### ISSUE TITLE
Wrong sprites- discord webhook posts

### ISSUE THREAD

**Issue by swimmy08** [LegoFigure11/RaidCrawler#117]
**Describe the bug**
I use the discord webhook feature. Since the major update, sprites have been consistently wrong on the discord posts that pop up when a match is found. 
Everything else is working normally, and the sprite I see on raid crawler is correct. 

**Screenshots**
![Screenshot 2023-01-24 091040](https://user-images.githubusercontent.com/123482653/214332743-225593c3-eb86-4cc4-861b-073ec7321e95.png)

**Comment by lfmundim** [comment]
I believe that uses PkHeX integration, it's probably a bug on their side rather

**Comment by jnovack** [comment]
I've seen quite a number of these, the GUI is correct with the Sprite (both in normal view, and the super awesome experimental view), it's only the webhook (which loads the file from the github repo) which is incorrect.

**Comment by lfmundim** [comment]
@jnovack i was running the app and got a similar scenario to OP's. 

<img width="1230" alt="image" src="https://user-images.githubusercontent.com/15164001/214378329-eb4187db-1d46-4c6b-92f7-bf586e2cef7b.png">

Doing some digging here is what I found:
* the encounter table right now is limited, as of course not all seeds have been found (or have they? is there something that ensures us?)
* on the screenshot I found Grafaiai. Grafaiai is Dex number 945. There is no 4 Star encounter for Species 945

**Comment by lfmundim** [comment]
To make quick work we might use a public base such as serebii
```cs
// need to append a 0 to the left of the number if <3 digits
url = $"https://www.serebii.net/scarletviolet/pokemon/new/{encounter.Species}.png"
```

**Comment by jnovack** [comment]
I'd rather do it once, and do it right.   But the issue here is that at some point, the raid numbers get out of sync with the national dex.  Grafaiai is 969 in the raid encounter tables, but 940 in the National Dex.

So, using things like serebii (is hotlinking permitted?) we need to pass 940, not 969.  Also, Serebii does have a Shiny nomenclature and form nomenclature.

**Comment by LegoFigure11** [comment]
Closed by [acad255](https://github.com/LegoFigure11/RaidCrawler/commit/acad2551991edc64ec26278ebc8dd3d6c6047258)
